analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -100,8 +100,10 @@ protected override void Initialize(SonarAnalysisContext context)
 
         private void Analyze(SyntaxNodeAnalysisContext context, CSharpSyntaxNode body, ISymbol symbol)
         {
+            var enabledAnalyzers = symbolicExecutionAnalyzerFactory.GetEnabledAnalyzers(context);
```

Comment:
this change would benefit of an issue + separate PR, as
- it's quite a nice catch
- it should be tracked in our issues 
- and the PR itself should be separated from the refactoring-only changes

Reply:
Done in #5120.

When you've pointed it out, it should actually be a quite impactful change, as we don't have SE rules on a test code. I originally considered it more like a corner case.